am2alertapi.py

```diff
@@ -8,13 +8,13 @@
 #  ALERTAPI_URL - URL for AlertAPI, not including path
 #  ALERT_ORGANIZATION - Service Now Organization Name
 
-from flask import Flask, Response, request, abort, jsonify
+from quart import Quart, Response, request, abort, jsonify
 from prometheus_client import multiprocess
 from prometheus_client import generate_latest, CollectorRegistry, CONTENT_TYPE_LATEST, Counter
+import asyncio
 import json
-import requests
+import httpx
```

Comment:
interesting, requests is old news now

Reply:
Only in the context of async. requests isn't implemented async aware.

I could have kept requests and not worried about the alertapi calls being async. But that's where it's desirable, ie where IO wait can occur.